edx_django_utils/monitoring/internal/code_owner/middleware.py

```diff
@@ -43,105 +41,85 @@ def process_exception(self, request, exception):
 
     def _set_code_owner_attribute(self, request):
         """
-        Sets the code_owner custom attribute, as well as several supporting custom attributes.
-
-        See CodeOwnerMonitoringMiddleware docstring for a complete list of attributes.
-
+        Sets the code_owner custom attribute for the request.
         """
-        code_owner, path_error = self._set_code_owner_attribute_from_path(request)
-        if code_owner:
-            set_custom_attribute('code_owner', code_owner)
-            return
-        if not path_error:
-            # module found, but mapping wasn't configured
-            code_owner = self._set_code_owner_attribute_catch_all()
+        module = self._get_module_from_request(request)
```

Comment:
@idegtiarov: This is some of the clean-up I had promised. Hopefully getting the module before getting the code_owner makes this code easier to understand.

Reply:
Good to know! Thanks.